### PR TITLE
FR-1542: Create General Application

### PR DESCRIPTION
**Pull request by shaunganley**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FR-1542

### Change description ###

Create General Application 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://finrem-cos-aat.service.core-compute-aat.internal",
+      "cosUrl": "http://finrem-cos-pr-450.service.core-compute-preview.internal",
```
Comment (shaunganley) [inline package.json:22]: to be reverted

**Review by ocarleton — APPROVED** [review]

**Review by johnoneill46 — APPROVED** [review]
Good shout on the non-prod naming convention. LGTM

**Review by HarryH96 — APPROVED** [review]

File: definitions/contested/json/State/State.json [hunk 2]
```diff
@@ -230,5 +230,13 @@
...
     "DisplayOrder": 29,
     "TitleDisplay": "**${[CASE_REFERENCE]}  ${applicantLName}  vs ${respondentLName}**"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "generalApplication",
+    "Name": "General Application",
+    "DisplayOrder": 30,
+    "TitleDisplay": "**${[CASE_REFERENCE]}  ${applicantLName}  vs ${respondentLName}**"
```
Comment (jakub-kozlowski) [inline definitions/contested/json/State/State.json:240]: spacing is inconsistent - 2x 2 spaces and then 1 space between `vs` and `respondentLName` - is this as expected?

Comment (shaunganley) [reply definitions/contested/json/State/State.json:240]: it's inconsistently consistent with every other state

**Pull request merged**